drivers/can/can_rcar.c

```diff
@@ -623,9 +623,9 @@ static int can_rcar_stop(const struct device *dev)
 
 	k_mutex_lock(&data->inst_mutex, K_FOREVER);
 
-	ret = can_rcar_enter_halt_mode(config);
+	ret = can_rcar_enter_reset_mode(config, true);
```

Comment:
Are the installed RX filters preserved when entering reset mode (does the `tests/drivers/can/api` test suite pass with this change)?